.github/actions/setup/action.yml

```diff
@@ -2,10 +2,10 @@ name: Setup
 runs:
   using: "composite"
   steps:
-    - name: Install Node.js 18.x
+    - name: Install Node.js 18.16.0
```

Comment:
Any reason for specific version? (not a blocker against the requested changes)

Reply:
Yeah, the default is 15 which contains an undici bug. Bumping to 16 makes the CI/CD pass. We'll go back to .X when we upgrade to Node 20